diy-omd.ino

```diff
@@ -6,6 +6,68 @@
 #include <DallasTemperature.h>
 #include <EEPROM.h>
 
+
+// Model types
+// RC Car ESC
+#define RC 0
+// Quadcopter ESC
+#define QUAD 1
+// Sidewinder 4 ESC
+#define SIDEWINDER 2
+
+
+// Set model here. No equals, just the value after ESC_TYPE
+#define ESC_TYPE SIDEWINDER
+
+
+#if ESC_TYPE == RC
+
+#define ESC_PIN1 2
+#define ESC_PIN2 11
+#define ESC_PIN3 24
+#define HALL_PIN 14
+
+#define STOP_VALUE 90
+#define REVERSE_VALUE 0
+#define ACC_VALUE 115
+#define CALIBRATE_VALUE 180
+#define OMD_GO_VALUE 135
+#define BALANCE_GO_VALUE 108
+#define MAX_GO_VALUE 130
+
+#elif ESC_TYPE == QUAD
+
+#define ESC_PIN1 2
+#define ESC_PIN2 11
+#define ESC_PIN3 24
+#define HALL_PIN 14
+
+#define STOP_VALUE 0
+#define REVERSE_VALUE 0
+#define ACC_VALUE 15
+#define CALIBRATE_VALUE 180
+#define OMD_GO_VALUE 27
+#define BALANCE_GO_VALUE 20
+#define MAX_GO_VALUE 60
+
+#elif ESC_TYPE == SIDEWINDER
+
+#define ESC_PIN1 37
+#define ESC_PIN2 15
+#define ESC_PIN3 14
```

Comment:
Why are your pins dependent on the ESC type? For us, we had a soldered proto board with headers that were soldered to fixed pins. Then switched the connector out only when changing to another ESC.

Reply:
I started the board before getting the code, so didn't know the pins.